go.mod

```diff
@@ -135,6 +134,9 @@ require (
 // When updating replace rules, make sure to also update the rules in integration/client/go.mod and api/go.mod
 replace (
 	github.com/gogo/googleapis => github.com/gogo/googleapis v1.3.2
+
+	// prevent go mod from rolling this back to the last tagged release
+	github.com/opencontainers/image-spec => github.com/opencontainers/image-spec v1.0.2-0.20211117181255-693428a734f5
```

Comment:
This looks like a possible bug in go modules; this is a direct dependency of containerd, but it ignores the version we specify, and rolls it back to the last tagged release